tools/unitctl/unit-client-rs/src/unit_client.rs

```diff
@@ -185,7 +185,14 @@ impl UnitClient {
     }
 
     pub fn new_http(control_socket: ControlSocket) -> Self {
-        let remote_client = Client::builder().build(HttpsConnector::new());
+        let remote_client = Client::builder()
+            .build(HttpsConnectorBuilder::default()
+                   .with_native_roots()
+                   .unwrap_or_else(|_| HttpsConnectorBuilder::default()
+                                   .with_webpki_roots())
+                   .https_or_http()
+                   .enable_http1()
```

Comment:
not a blocker, more of a curiosity question: is [http2 something we might want to enable](https://docs.rs/hyper-rustls/latest/hyper_rustls/struct.HttpsConnectorBuilder.html#method.enable_http2-1) here as well?

Reply:
The only case where that would be used that I can imagine is in the case of a load balancer that is set up to serve HTTP2 to clients, but to relay HTTP1 back to the Unit control API. 

Since its easy and we dont have to support the logic directly Ill enable it, but it is a funky little corner case.